doc/api/tls.md

```diff
@@ -357,15 +357,15 @@ the default configuration. If these clients _must_ be supported, the
 [TLS recommendations][] may offer a compatible cipher suite. For more details
 on the format, see the OpenSSL [cipher list format][] documentation.
```

Comment:
Not related to this PR necessarily, but this paragraph apparently is for people who need compatibility with clients like IE6. I wonder if we can/should remove it.